go.mod

```diff
@@ -6,22 +6,25 @@ require (
 	cloud.google.com/go v0.45.1
 	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190612171043-2e19bb35a278
 	github.com/container-storage-interface/spec v1.2.0
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.3.4
 	github.com/google/uuid v1.1.1
 	github.com/hashicorp/go-multierror v1.0.0 // indirect
+	github.com/kubernetes-csi/csi-proxy/client v0.0.0-20200319061913-d6ab31300107
```

Comment:
We can actually use the real tag version, but we need add a special tag to the csi-proxy repo

Reply:
I will work on it.